src/main/resources/org/jenkinsci/plugins/nomad/NomadSlaveTemplate/config.jelly

```diff
@@ -16,6 +16,10 @@
             <f:textbox/>
         </f:entry>
 
+        <f:entry title="${Constraints}" field="constraints" description="comma separated LTarget,RTarget,Operand">
```

Comment:
using comma may interfere with `set_contains` with assume `RTarget` is a comma separated list of values